src/lib/swapper/swappers/CowSwapper/CowSwapper.test.ts

```diff
@@ -66,7 +66,7 @@ describe('CowSwapper', () => {
 
   describe('getType', () => {
     it('returns the correct type for CowSwapper', () => {
```

Comment:
And add test for the Gnosis flavor
```suggestion
    it('returns the correct type for Ethereum CowSwapper', () => {
```